client-cli/src/main.rs

```diff
@@ -60,14 +60,19 @@ pub(crate) fn chain_id() -> Option<String> {
     std::env::var("CRYPTO_CHAIN_ID").map(Some).unwrap_or(None)
 }
 
-#[inline]
+#[cfg(not(test))]
 pub(crate) fn ask_passphrase(message: Option<&str>) -> Result<SecUtf8> {
     ask(message.unwrap_or("Enter passphrase: "));
     password()
         .map(Into::into)
         .chain(|| (ErrorKind::IoError, "Unable to read password"))
 }
 
+#[cfg(test)]
+pub(crate) fn ask_passphrase(_message: Option<&str>) -> Result<SecUtf8> {
+    Ok(SecUtf8::from("123456"))
+}
+
```

Comment:
I'm not sure if this helps with testing, as it'll skip the actual code to be tested and have a replacement -- most things in client are tested in isolation.
For cli testing, it's a bit tricky, but ideally it'd be done with passing in custom streams instead of using stdin/stdout

Reply:
could you make entering value via stream as separate issue?
so we can unit-test for those functions.